app/assets/stylesheets/questions.css.scss

```diff
@@ -1,48 +1,56 @@
-.all-mine-other-filter {
-  margin-right: 25px;  // aligns with right side of question list
-}
-
-// (All-mine-filter relocation at different screen sizes)
-// On small screens (bootstrap xs) show all-mine-other filter
-// just above the question list, since it wraps onto its own .row
-#all-mine-filter-container-xs {
+// On small screens (bootstrap xs) show-small-screen and replace with show-large-screen otherwise
+#show-small-screen {
   display: none;
 }
 @media only screen and (max-width: 767px) {
-  #all-mine-filter-container-xs {
+  #show-small-screen {
     display: block;
   }
-  #all-mine-filter-container {
+  #show-large-screen {
     display: none;
   }
 }
-// End of (All-mine-filter relocation at different screen sizes)
 
-.generate-question-button-filter {
-  margin-left: 12px;  // aligns with left side of question list
-}
 
-// (generate-question-button-filter relocation at different screen sizes)
-// On small screens (bootstrap xs) show generate-question-button-filter
-// just above the question list, since it wraps onto its own .row
-#generate-question-button-filter-xs {
-  display: none;
-}
 @media only screen and (max-width: 767px) {
-  #generate-question-button-filter-xs {
-    display: block;
+  
+  #all-mine-other-group {
+    float: none !important;
```

Comment:
Why is this `float` `!important` stuff needed? maybe a comment here would help, if it is needed, for people like me who aren't real CSS-savvy.